vizro-ai/docs/pages/explanation/faq.md

```diff
@@ -16,3 +16,13 @@
 [Petar Pejovic](https://github.com/petar-qb).
 
 With thanks to Sam Bourton and Stephen Xu for sponsorship, inspiration and guidance, plus everyone else who helped to test, guide, support and encourage development.
+
```

Comment:
I would prefer not to make this addition but instead point to the page to where this moved, under the subheading "Supported models" https://vizro.readthedocs.io/projects/vizro-ai/en/latest/pages/user-guides/customize-vizro-ai/#supported-models

Reply:
sounds good, just commenting on the other option i am thinking, thank you